shared/build.gradle

```
@@ -15,4 +16,5 @@ dependencies {
     implementation 'org.jetbrains.kotlin:kotlin-stdlib-jdk8'
     testImplementation 'org.hamcrest:hamcrest-all:1.3'
 	testImplementation 'junit:junit:4.11'
+    implementation("org.jetbrains.kotlinx:kotlinx-serialization-json:1.2.2")
```

I know this isn't part of the PR, but while we're at it, could you fix the indentation here?

```suggestion
    testImplementation 'org.hamcrest:hamcrest-all:1.3'
    testImplementation 'junit:junit:4.11'
    implementation("org.jetbrains.kotlinx:kotlinx-serialization-json:1.2.2")
```